ActionButton mvc

jcaliksygic opened the pull request:
deleted viewModel,
actionButton is configured by its properties

jcaliksygic committed:
ActionButton mvc

deleted viewModel,
actionButton is configured by its properties

jcaliksygic committed:
CI test

Commit:
Try to fix travis ci build

Commit:
Specify another travis ci simulator

Commit:
Merge branch 'feature/mvc_switch' into feature/action-button-mvc

# Conflicts:
#	.travis.yml
#	SygicUIKit.podspec

jcaliksygic committed:
Merge branch 'feature/poi-detail-mvc' of github.com:Sygic/sygic-ui-kit-ios into feature/action-button-mvc

Kraci commented on SygicUIKit/Classes/Views/ActionButton/SYUIActionButton.swift:
```diff
@@ -42,15 +42,62 @@ public enum SYUIActionButtonSize: CGFloat {
     public var action: (() -> Void)?
-    public var title = UILabel()
-    public var subtitle = UILabel()
+    private var titleLab = UILabel()
```
```suggestion
    private var titleLabel = UILabel()
```

jcaliksygic committed:
ActionButton

public titleLabel, subtitleLabel that reference private customTitleLabel and customSubtitleLabel,
cleanup

Kraci reviewed (approved):
tip top 👍
